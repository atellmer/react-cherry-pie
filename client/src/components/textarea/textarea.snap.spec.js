import React from 'react';
import renderer from 'react-test-renderer';

import TmTextarea from './index';


jest.mock('react-dom');
jest.mock('react-textarea-autosize', () => 'TextareaAutosize');

test('Jest: TmTextarea (Snapshot)', () => {
  const props = {
    value: 'lorem ipsum',
    onChangeInput: (value) => {},
    onHeightChange: () => {}
  };

  const component = renderer.create(<TmTextarea {...props}/>);

  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
