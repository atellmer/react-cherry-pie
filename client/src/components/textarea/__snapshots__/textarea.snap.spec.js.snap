exports[`test Jest: TmTextarea (Snapshot) 1`] = `
<TextareaAutosize
  className={undefined}
  maxRows={10}
  minRows={1}
  onChange={[Function]}
  onHeightChange={[Function]}
  placeholder="Введите ваше сообщение..."
  value="lorem ipsum" />
`;
